### ISSUE TITLE
0.59 doesn't start on Fedora 33

### ISSUE THREAD

**Issue by laurent-indermuehle** [sedwards2009/extraterm#342]
I've upgraded Extraterm and extraterm-commands from 0.58 to 0.59. I could start the program then. But after reboot starting extraterm from gnome-shell does nothing. Starting extraterm from a terminal yield:

```
panic: Post "http://127.0.0.1:41349/039578a5116422aac8af573edfade1b8/command": dial tcp 127.0.0.1:41349: connect: connection refused

goroutine 1 [running]:
main.runCommand(0xc0000c0106, 0x37, 0xc0000bdf08, 0x13, 0xc0000d83e8, 0x0)
        /home/runner/work/extraterm-launcher/extraterm-launcher/cmd/extraterm-launcher/extraterm-launcher.go:196 +0x2f3
main.runOpenWindowAtCommand(0xc0000c0106, 0x37, 0xc0000160e4, 0x18, 0x401230)
        /home/runner/work/extraterm-launcher/extraterm-launcher/cmd/extraterm-launcher/extraterm-launcher.go:131 +0xf7
main.main()
        /home/runner/work/extraterm-launcher/extraterm-launcher/cmd/extraterm-launcher/extraterm-launcher.go:40 +0xb6
```
Any idea why it worked only once? What is that /home/runner ?

I've roll back to 0.58 and it worked normally now.

**Comment by sedwards2009** [comment]
Delete the file `~/.config/extraterm/ipc.run` and it will start up again.

I noticed bug earlier this week too and am working on a solution. This is part of the system for re-using already running Extraterm instances.

(`/home/runner/` is from where the executable was built.)

I'll do a bug fix release soon.

**Comment by laurent-indermuehle** [comment]
Great! In that case, I stay in 0.58. This way I'm ready to validate that the hotfix works.
Thanks for your work!

**Comment by sedwards2009** [comment]
Fixed in v0.59.1.

**Comment by laurent-indermuehle** [comment]
v0.59.1 works like a charm! Thanks for the fix!
